Day-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,14 @@
 import React from 'react';
 import './Card.css';
+
+const CardContent = () => {
+    return (
+        <div className="card-text">
+            <h3>Get started with React</h3>
```

Comment:
After passing the props into the component in the review comment I left in your CardContainer on line 10. You would use the props here like `<h3>{props.h3Text}</h3>` and so on.